lib/charms/mysql/v0/mysql.py

```diff
@@ -116,15 +116,17 @@ def wait_until_mysql_connection(self) -> None:
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 49
+LIBPATCH = 50
```

Comment:
please update to 51 as another PR was merged with libpatch 50

Reply:
096fe7f2